git.adoc

```diff
@@ -10,35 +10,33 @@ Sharing some git conveniences `~/.bashrc` aliases and scripts.
 +
 [source,text]
 ----
-#!/usr/bin/bash
+#!/usr/bin/env bash
 
-cd ~/openshift-docs
```

Comment:
The assumption is that you run this from your OpenShift repo. (The script checks at least if it's in a Git repo) I find hardcoding this path a very bad practice.